ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -226,7 +227,7 @@ void RSkComponentTextInput::processEventKey (rnsKey eventKeyType,bool* stopPropa
   auto textInputEventEmitter = std::static_pointer_cast<TextInputEventEmitter const>(component.eventEmitter);
   auto const &textInputProps = *std::static_pointer_cast<TextInputProps const>(component.props);
   keyPressMetrics.text = RNSKeyMap[eventKeyType];
-    if ((eventKeyType >= RNS_KEY_1 && eventKeyType <= RNS_KEY_z)) {
+    if ((eventKeyType >= RNS_KEY_1 && eventKeyType <= RNS_KEY_Less)) {
```

Comment:
Can we have  Key_Type_Start & Key_TYPE_End enum to confine to pack the keys in groups .
Ex:-RNS_RCU_KEY_START,RCU_KEY_END,RNS_NAV_KEY_STRAT,RNS_NAV_KEY_END.
I feel, Which may give flexibility to modify the key support in future .

Reply:
discussed offline, unresolved